lib/ansible/modules/cloud/amazon/ec2_tag.py

```diff
@@ -110,6 +117,15 @@
 - name: List tags, such as Name and env
   debug:
     msg: '{{ ec2_tags.tags.Name }} {{ ec2_tags.tags.env }}'
+
+- name: Remove tag by key only
+  ec2_tag:
+    region: eu-west-1
+    resource: vol-XXXXXX
+    state: absent
+    purge_tags: yes
```

Comment:
I haven't tested what this code change does exactly, but I think this example is off. Purge_tags is used with state: present, because it usually means 'remove any tags except these which should be present'.